build.psm1

```diff
@@ -1410,6 +1410,11 @@ function Start-PSPackage {
         "AppImage" {
             if ($IsUbuntu14) {
                 Start-NativeExecution { bash -iex "$PSScriptRoot/tools/appimage.sh" }
+                $appImage = Get-Item PowerShell-*.AppImage
+                if ($appImage.Count -gt 1) {
+                    Write-Error "Found more than one AppImage package"
+                }
+                Rename-Item $appImage.Name $appImage.Name.Replace("-","-$Version.")
```

Comment:
Does AppImage itself have any assumption of its name? I mean, if we just rename the file, would it cause any problem when running the AppImage? I wonder if we can do the same for the OSX package as well (#3058)